content/en/docs/ops/deployment/security-model/index.md

```diff
@@ -0,0 +1,218 @@
+* What the impact is (total remote execution, denial of service, etc).
+
+In this document, we will primarily consider the worst case scenario: a compromised component means an attacker has complete remote code execution capabilities.
+
+### Workload compromise
+
+In this scenario, an application workload (Pod) is compromised.
+
+A pod [*may* have access](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#opt-out-of-api-credential-automounting) to its service account token.
+If so, a workload compromise can move laterally from a single Pod to compromising the entire service account.
+
+{{< tabset category-name="dataplane" >}}
+{{< tab name="Sidecar mode" category-value="sidecar" >}}
+In the sidecar model, the proxy is co-located with the pod, and runs within the same trust boundary.
+A compromised application can tamper with the proxy through the admin API or other surfaces.
+It should be assumed that a compromised workload also includes a compromise of the sidecar proxy.
+
+Given this, a compromised workload may:
+* Send arbitrary traffic, with or without mutual TLS.
+  These may bypass any proxy configuration, or even the proxy entirely.
+  Note that Istio does not offer Egress-based authorization policies, so there is no authorization policy bypass occurring.
+* Accept traffic that was already destined to the application. It may bypass policies that were configured in the sidecar proxy.
+
+The key takeaway here is that while the compromised workload may behave maliciously, this does not give them the ability to bypass policies in _other_ workloads.
+{{< /tab >}}
+
+{{< tab name="Ambient mode" category-value="ambient" >}}
+In ambient mode, the node proxy is not co-located within the pod, and runs in another trust boundary as part of an independent pod.
+
+A compromised application may send arbitrary traffic.
+However, they do not have control over the node proxy, which will chose how to handle incoming and outbound traffic.
+
+Additionally, as the pod itself doesn't have access to a service account token to request a mutual TLS certificate, lateral movement possibilities are reduced.
+{{< /tab >}}
+{{< /tabset >}}
+
+Istio offers a variety of features that can limit the impact of such a compromise:
+* [Observability](/docs/tasks/observability/) features can be used to identify the attack.
+* [Policies](/docs/tasks/security/authorization/) can be used to restrict what type of traffic a workload can send or receive.
+
+### Proxy compromise - Sidecars
+
+In this scenario, a sidecar proxy is compromised.
+Because the sidecar and application reside in the same trust domain, this is functionally equivilent to the [Workload compromise](#workload-compromise).
+
+### Proxy compromise - Waypoint
+
+In this scenario, a [waypoint proxy](#gateways-and-waypoints) is compromised.
+While waypoints do not have any privileges for a hacker to exploit, they do serve (potentially) many different services and workloads.
```

Comment:
unsandwiched waypoint have the private key which identifies them, this can be exfiltrated to enable an external attacker. Sandwiched ones do not

Reply:
Also note that a waypoint _never_ shares its identity with any application so cannot impersonate one when calling other services